'FEValuesViews': the symbol to the left of a '::' must be a type  (MSVC)

I guess  the error caused by the namespace conflict( such as dealii::FEValuesViews and dealii::internal::FEValuesViews) because of error C2039('ViewType': ...)
> Severity	Code	Description	Project	File	Line	Suppression State
Error	C2938	'dealii::FEValuesViews::View<dim,spacedim,Extractor>' : Failed to specialize alias template	obj_meshworker_debug	L:\compile\dealii\include\deal.II\meshworker\scratch_data.h	45	
Error	C3083	'FEValuesViews': the symbol to the left of a '::' must be a type	obj_meshworker_debug	L:\compile\dealii\include\deal.II\fe\fe_values.h	1976	
Error	C2631	'ScratchData': a class or enum cannot be defined in an alias template	obj_meshworker_debug	L:\compile\dealii\include\deal.II\meshworker\scratch_data.h	230	
Error	C2039	'ViewType': is not a member of 'dealii::FEValuesViews::internal'	obj_meshworker_debug	L:\compile\dealii\include\deal.II\fe\fe_values.h	1976	
Error	C2059	syntax error: '<L_TEMPLATEDECL>'	obj_meshworker_debug	L:\compile\dealii\include\deal.II\meshworker\scratch_data.h	1262	
Error	C2059	syntax error: '}'	obj_meshworker_debug	L:\compile\dealii\source\meshworker\scratch_data.cc	1	
Error	C2061	syntax error: identifier 'ViewType'	obj_meshworker_debug	L:\compile\dealii\include\deal.II\fe\fe_values.h	1976	
Error	C2143	syntax error: missing ';' before '}'	obj_meshworker_debug	L:\compile\dealii\include\deal.II\meshworker\scratch_data.h	47	
Error	C2143	syntax error: missing ';' before '}'	obj_meshworker_debug	L:\compile\dealii\source\meshworker\scratch_data.cc	1	
Error	C2062	type 'unknown-type' unexpected	obj_meshworker_debug	L:\compile\dealii\include\deal.II\meshworker\scratch_data.h	45	
Error	C1903	unable to recover from previous error(s); stopping compilation	obj_meshworker_debug	L:\compile\dealii\source\meshworker\scratch_data.cc	1	
Error	C2988	unrecognizable template declaration/definition	obj_meshworker_debug	L:\compile\dealii\source\meshworker\scratch_data.cc	1

--- a/include/deal.II/fe/fe_values.h
+++ b/include/deal.II/fe/fe_values.h
@@ -1972,8 +1972,8 @@ namespace FEValuesViews
    * @author Luca Heltai, 2019.
    */
   template <int dim, int spacedim, typename Extractor>
-  using View =
-    typename internal::FEValuesViews::ViewType<dim, spacedim, Extractor>::type;
+  using View = typename dealii::internal::FEValuesViews::
+    ViewType<dim, spacedim, Extractor>::type;
 } // namespace FEValuesViews
 
 

